src/main/java/roomescape/model/Theme.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.model;
+
+public class Theme {
+    private Long id;
+    private String name;
+    private String description;
+    private String thumbnail;
+
+    public Theme(String name, String description, String thumbnail) {
+        this.name = name;
+        this.description = description;
+        this.thumbnail = thumbnail;
+    }
```

Comment:
```suggestion
    public Theme(String name, String description, String thumbnail) {
        this(null, name, description, thumbnail);
    }
```

존재하는 주생성자를 활용해볼 수도 있을 것 같네요!
다른 생성자도 마찬가지구요 :)